pyproject.toml

```
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["oldest-supported-numpy", "setuptools >= 38.6", "pip >= 10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "conjugate_map"
+version = "0.0.2"
```

```suggestion
version = "0.2.0"
```

Changing version based on versioning of initial release.